### PR TITLE
 Add note regarding export of nested folders with Apple Mail

### PR DESCRIPTION
**Pull request by junikowski**
Per eesheesh's suggestion in #36

### CODE REVIEW

File: import-mailbox-to-gmail.py [hunk 1]
```diff
@@ -182,7 +182,8 @@ def process_mbox_files(username, service, labels):
   for root, dirs, files in os.walk(base_path):
     for dir in dirs:
       try:
-        get_label_id_from_name(service, username, labels, dir)
+        labelname = os.path.join(root[len(base_path) + 1:], dir)
```
Comment (eesheesh) [inline import-mailbox-to-gmail.py]: I think you need to sync your branch - this change is already in the master branch: https://github.com/google/import-mailbox-to-gmail/blob/85fc8afe0480c3323026504c5dedb7d6f6c5fd40/import-mailbox-to-gmail.py#L185

**Comment by junikowski** [comment]
Not sure what happened. I was supposing this was some just GitHub quirk. Will look into this & apologies for the extra work for you! :)

**Comment by eesheesh** [comment]
No problem! :) Let me know when I should take another look.

**Comment by junikowski** [comment]
Looks like [rebasing](https://stackoverflow.com/a/45286781) did the trick! Should only contain the right commit now.

**Comment by eesheesh** [comment]
Thanks! :)

**Pull request merged**